packages/data/package.json

```diff
@@ -35,8 +35,10 @@
     "ember-cli-htmlbars": "^4.2.2",
     "ember-cli-mirage": "~1.1.6",
     "ember-cli-mirage-graphql": "^0.3.1",
+    "ember-cli-moment-shim": "^3.8.0",
```

Comment:
Do we need this now that we can just import ember?

Reply:
When I didn't include the shim, it wasn't able to find the moment module. Is there some other configuration for ember-moment to include the ES6 module? I wasn't able to find anything about it

Reply:
ember-auto-import should be able to do it.

Reply:
<img width="594" alt="Screen Shot 2020-06-22 at 3 44 41 PM" src="https://user-images.githubusercontent.com/23023478/85333617-8147fd80-b49f-11ea-99ff-7b40aa3e96c5.png">
Removing the shim results in this even with ember-auto-import

Reply:
Did you install moment by itself?

Reply:
For some reason I thought we needed ember-moment here, but we don't use the template helpers so we can just use `moment`.